src/compiler/transformer.ts

```diff
@@ -71,11 +82,44 @@ namespace ts {
             transformers.push(transformES5);
         }
 
-        addRange(transformers, customTransformers && customTransformers.after);
+        addRange(transformers, customTransformers && map(customTransformers.after, wrapScriptTransformerFactory));
+        return transformers;
+    }
 
+    function getDeclarationTransformers(customTransformers?: CustomTransformers) {
+        const transformers: TransformerFactory<SourceFile | Bundle>[] = [];
+        transformers.push(transformDeclarations);
+        addRange(transformers, customTransformers && map(customTransformers.afterDeclarations, wrapDeclarationTransformerFactory));
         return transformers;
     }
 
+    /**
+     * Wrap a custom script or declaration transformer object in a `Transformer` callback with fallback support for transforming bundles.
+     */
+    function wrapCustomTransformer(transformer: CustomTransformer | CustomTransformer): Transformer<Bundle | SourceFile> {

```

Comment:
`CustomTransformer | CustomTransformer`?

Reply:
Typo due to refactoring...